### DISCUSSION TITLE
Provide in preview when uploading custom background.

### DISCUSSION THREAD

**Discussion by somaamos** [JulianPrieber/littlelink-custom#520] [Feature requests]
Usability enhancement idea: 
It would be nice if the Preview and/or Custom Background image on the Themes tab had visible "guide lines" showing how the site will render on both 16:9 and 9:16 (and/or other aspect ratios) so you can get a sense of whether the background image you've uploaded is going to work correctly before clicking Apply.

**Comment by JulianPrieber** [comment]
Each theme handles this differently
